models/lpjguess/inst/pecan.ins

```diff
@@ -133,7 +133,7 @@ run_pasture 0	      ! whether to simulate pasture (1) or not (0)
 run_natural 1	      ! whether to simulate natural vegetation (1) or not (0)
 run_peatland 1	      ! whether to simulate peatland (1) or not (0)
 
-@@@@@ Remove in PalEON version @@@@@
+! Remove in PalEON version 
```

Comment:
not that write.config is looking for the tag "##### Remove in PalEON version #####" not "! Remove in PalEON version" so I don't think these will be removed